src/js/live-tracker.js

```diff
@@ -167,24 +192,22 @@ class LiveTracker extends Component {
    * their initial value.
    */
   reset_() {
+    this.lastTime_ = null;
     this.pastSeekEnd_ = 0;
     this.lastSeekEnd_ = null;
```

Comment:
should we set this to `-1` or something instead of null to designate a turned off value? I would presume that -1 isn't a valid value for `lastSeekEnd_`. Then, below we can just check if it's -1 instead of whether it's a number.
```suggestion
    this.lastSeekEnd_ = -1;
```

Reply:
Yeah it's probably best to keep it a consistent type too.